CHANGELOG.md

```diff
@@ -1,8 +1,10 @@
 # Lotus changelog
 
 # UNRELEASED
+ (`ChainIndexer`) to index Filecoin chain state such as tipsets, messages, events and ETH transactions for accurate and faster RPC responses. The `ChainIndexer` replaces the existing `MsgIndex`, `EthTxIndex` and `EventIndex` implementations in Lotus, which [suffer from a multitude of known problems](https://github.com/filecoin-project/lotus/issues/12293).  If you are an RPC provider/node operator, please refer to the [ChainIndexer documentation for RPC providers](TODO: URL) for information on how to enable, configure and use the new Indexer.
```

Comment:
I just learnt we are introducing a HUGE breaking change here where the database migration is is mandatory. it should be called out LOUDER with ⚠️ ⚠️  here + in the release note!

Reply:
```suggestion
 (`ChainIndexer`) to index Filecoin chain state such as tipsets, messages, events and ETH transactions for accurate and faster RPC responses. The `ChainIndexer` replaces the existing `MsgIndex`, `EthTxHashLookup` and `EventIndex` implementations in Lotus, which [suffer from a multitude of known problems](https://github.com/filecoin-project/lotus/issues/12293).  If you are an RPC provider/node operator, please refer to the [ChainIndexer documentation for RPC providers](TODO: URL) for information on how to enable, configure and use the new Indexer.  While there is no migration and one can upgrade and downgrade without backups, there are manual steps that need to be taken to backfill data for the new Indexer.
```

Given discussion below do we still consider this a HUGE breaking change since there is no migration per-se (but rather backfilling).  I have taken a stab at some extra language.

Reply:
Handled in https://github.com/filecoin-project/lotus/pull/12600